### PR TITLE
alignment with architecture document

### PR DESCRIPTION
**Pull request by louberger**

### CODE REVIEW

File: draft-ietf-detnet-dp-sol-mpls-02.xml [hunk 1]
```diff
@@ -120,8 +119,8 @@ LB: General comments:
 
   <abstract>
    <t>
-    This document specifies Deterministic Networking data plane encapsulation solutions. The
-    described data plane solutions is applied over an MPLS Packet Switched Networks.
+    This document specifies Deterministic Networking operation
```
Comment (vb-balazs) [inline draft-ietf-detnet-dp-sol-mpls-02.xml]: operation means also control and management plane. We cover data plane only here, with some hints to CP and MP.

**Comment by louberger** [comment]
I mean operation as in behavior, not OAM.

File: draft-ietf-detnet-dp-sol-mpls-02.xml [hunk 2]
```diff
@@ -351,44 +329,46 @@ End System      Node            Node          Node         End System
...
+
+        |<- DN IP->| |<---- DetNet MPLS ---->| |< -DN IP ->|
         ]]></artwork>
   </figure>
   <t>
    <xref target="fig_ip_mpls_detnet"/> and <xref
    target="fig_8021_detnet"/>, show different cases where relay nodes
    may be used.  Relay nodes are similar to edge nodes in that both are
-   aware of the needs of particular DetNet flows and take care to
+   aware of the needs of particular DetNet App-flows and take care to
```
Comment (vb-balazs) [inline draft-ietf-detnet-dp-sol-mpls-02.xml]: Relay nodes know only DetNet flows, not App-flows.

File: draft-ietf-detnet-dp-sol-mpls-02.xml [hunk 3]
```diff
@@ -644,7 +624,7 @@ CE1----EN1--------R1-------R2-------R3--------EN2-----CE2
         <vspace blankLines="1"/>
         Generally, distributing packets of the same DetNet flow over
         multiple paths is not recommended. Such load sharing,
-        e.g., via ECMP or UCMP, impacts ordering and end-to-end jitter.
+        e.g., via ECMP or UCMP, impacts ordering and effective jitter.
```
Comment (vb-balazs) [inline draft-ietf-detnet-dp-sol-mpls-02.xml]: Is "effective jitter" a new term?

**Pull request merged**